lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -537,6 +568,13 @@ def _create_domain_and_attach_logger(
         # Start the VM.
         node_context.domain.resume()
 
+        # Once libvirt domain is created, check if driver attached to device
```

Comment:
Add `if node_context.is_device_passthrough_set:` check here to match the `ch_platform.py`'s code?

Reply:
@cwize1 We have removed `is_device_passthrough_set` now.

Reply:
@cwize1 Please check the latest push.